core/src/main/kotlin/org/opensearch/alerting/core/settings/AlertingV2Settings.kt

```diff
@@ -0,0 +1,17 @@
+package org.opensearch.alerting.core.settings
+
+import org.opensearch.common.settings.Setting
+
+/**
+ *  This class exclusively houses the Alerting V2 enabled setting, so that both Monitor V2 Stats
+ *  and the rest of the CRUD APIs can read it
+ */
+class AlertingV2Settings {
+    companion object {
+        val ALERTING_V2_ENABLED = Setting.boolSetting(
+            "plugins.alerting_v2.enabled",
```

Comment:
@lezzago do you reckon it's ok to introduce a new prefix `plugins.alerting_v2` rather than `plugins.alerting.v2` since it's housed in alerting itself?